components/DropdownLight/DropdownLight.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { colors, spacing, baseFontSize } from '../shared/theme';
+
+const itemPropType = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.shape({
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    label: PropTypes.string,
+  }),
+]);
+
+const DropdownSelect = styled.select`
+  width: 100%;
+  height: 44px;
+
+  ${({ theme }) => {
+    const { baseFontSize: baseFontSizeDropdownSelect } = theme;
+
+    return css`
+      font-size: ${baseFontSizeDropdownSelect}px;
+    `;
+  }}
+`;
+
+const DropdownLight = ({ disabled, items, theme }) => (
+  <>
+    <DropdownSelect disabled={disabled} theme={theme}>
+      {items.map(item => (
+        <option key={items.value || items}>{item.label || item}</option>
```

Comment:
You are using the entire array as a key instead of using just one array item. Also the "value" attribute is not being set in the "option" tag

Reply:
I made a mistake, I put the key in the plural